src/mlpack/methods/random_forest/random_forest_impl.hpp

```diff
@@ -18,6 +18,25 @@
 namespace mlpack {
 namespace tree {
 
+template<
+    typename FitnessFunction,
+    typename DimensionSelectionType,
+    template<typename> class NumericSplitType,
+    template<typename> class CategoricalSplitType,
+    typename ElemType
```

Comment:
`ElemType` was removed in https://github.com/mlpack/mlpack/pull/2874, can you merge in the latest master?